style.css

```diff
@@ -0,0 +1,6 @@
+h1 {
+  color: #00ff;
+  font-size: 50px;
+  display: flex;
+  justify-content: center;
```

Comment:
- [ ] You have a linter error on this project. Kindly fix the error. :+1:

Reply:
Hello Assalamualaikum. can you please check it out again now? I've corrected some errors that you mentioned. Thanks.